main.go

```diff
@@ -8,11 +8,11 @@ import (
 	log "github.com/Sirupsen/logrus"
```

Comment:
Lets try and seperate your PRs. If you are making changes to services open a PR just for the services and a seperate one for the UI.